### PR TITLE
[CalendarV2] fix bugs

### LINKED ISSUE #352: Test/Bug: No suggestions due to timezones
Discovered by booking time with L (Seattle) and D (London):
- When there is working hour overlap (London & Amsterdam) - I get suggestions : known bug that card is too large, Yue working on it #340 
- No overlap (Seattle & Amsterdam) - I get none - we skip the suggestion step - so can we add transcript here (I've added to the transcript file):
**"I can't suggest a time to meet since you have no overlap in working hours, what day did you have in mind?"
"You seem to have no overlap in working hours, what day did you have in mind?"**

image

### LINKED ISSUE #364: Test/Bug: Asking about next week/month no response
4. Asking about next week/month no response, weekend works great (with carrousel for saturday/sunday)
image
image

In the design & transcript file we have different utterances and logic for:

weekend = show saturday/sunday in carrousel (that works in Teams load)
For the weekend, I see you have [#] events:
I see you have [#] events planned in the weekend:
Shows card for saturday/sunday

week/month = show just the upcoming Monday + additional transcript:
Next week, I see you have [#] events. First up on Monday:
Next month, I see you have [#] events. First up on Monday:
Shows card for monday

image

### LINKED ISSUE #378: Test/Bug: Can not find specific meeting when there are lots of meetings before it
I have a meeting named "plan".
When I set this meeting on Nov.16, I can find it. But if I move it to Nov.20, I cann't find it.
image
image

### LINKED ISSUE #380: Test/Bug: use alterations in response to cancel/quit/stop
Please use the following as in the transcript Excel, excluding the: "let me know I can help.."
- OK, no problem.
- Sure. I'll cancel.
- OK, I will stop.
- Got it, I'll stop here.
- OK, no problem. I know I'm not perfect yet.
image

### PR DESCRIPTION
**Pull request by xhr0804**
### Purpose
*What is the context of this pull request? Why is it being done?*
Close #378 
close #364 
close #352 
close #380

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg [hunk 1]
```diff
@@ -71,10 +71,19 @@
 - And at what time?
 - Got it, and when do you want it to start?
 
+# AskForTimePrompt(noSuggestion)
+- IF: ${ noSuggestion == true }
+    - I can't suggest a time to meet, probably because there is no overlap in working hours. What day did you want to meet?
+    - There's aren't any relevant suggestions, probably there is no overlap in working hours. What day did you have in mind?
+    - I couldn't find relevant suggestions that works for all attendees. What day did you want to meet?
```
Comment (lauren-mills) [inline skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg:78]: I couldn't find relevant suggestions that work for all attendees. What day did you want to meet?

File: skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg [hunk 2]
```diff
@@ -71,10 +71,19 @@
 - And at what time?
 - Got it, and when do you want it to start?
 
+# AskForTimePrompt(noSuggestion)
+- IF: ${ noSuggestion == true }
+    - I can't suggest a time to meet, probably because there is no overlap in working hours. What day did you want to meet?
+    - There's aren't any relevant suggestions, probably there is no overlap in working hours. What day did you have in mind?
```
Comment (lauren-mills) [inline skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg:77]: We should remove this one. The wording is not quite right.

File: skills/declarative/calendarSkill/dialogs/GetScheduleDialog/GetScheduleDialog.dialog [hunk 3]
```diff
@@ -84,15 +84,33 @@
                   "$designer": {
                     "id": "RgCy25"
                   },
-                  "resultProperty": "dialog.events",
+                  "resultProperty": "dialog.groups",
```
Comment (lauren-mills) [inline skills/declarative/calendarSkill/dialogs/GetScheduleDialog/GetScheduleDialog.dialog:87]: dialog.groupedEvents would be easier to understand here

**Review by lauren-mills — APPROVED** [review]

File: skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg (hunk 1 above)
Comment (xhr0804) [reply skills/declarative/calendarSkill/dialogs/CreateEventDialog/language-generation/en-us/CreateEventDialog.en-us.lg]: fixed

**Pull request merged**